### PR TITLE
Story/cite 135

### PR DESCRIPTION
**Pull request by MaulikLim**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
(you can copy the ticket if it hasn't changed) 

[CITE-135] - Search author authority pop needs current group tab
  
## Anything else the reviewer needs to know?

... describe here ...

[CITE-135]: https://diging.atlassian.net/browse/CITE-135

### CODE REVIEW

File: citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityEntryController.java [hunk 1]
```diff
@@ -63,6 +64,23 @@
...
     
+    @RequestMapping("/auth/authority/{zoteroGroupId}/find/authorities/group")
+    public ResponseEntity<AuthoritySearchResult> getGroupAuthorities(@PathVariable("zoteroGroupId") String zoteroGroupId,
+            @RequestParam(defaultValue = "0", required = false, value = "page") int page,
+            @RequestParam(defaultValue = "10", required = false, value = "pageSize") int pageSize,
+            @RequestParam("firstName") String firstName, @RequestParam("lastName") String lastName) {
+        
+        AuthoritySearchResult authorityResult = new AuthoritySearchResult();
+
+        Page<IAuthorityEntry> userEntries = authorityService.findByGroupAndName(Long.valueOf(zoteroGroupId),
```
Comment (jdamerow) [inline citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityEntryController.java:75]: why is this variable called `userEntries`?

File: citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityEntryController.java [hunk 2]
```diff
@@ -63,6 +64,23 @@
         return new ResponseEntity<AuthoritySearchResult>(authorityResult, HttpStatus.OK);
     }
     
+    @RequestMapping("/auth/authority/{zoteroGroupId}/find/authorities/group")
+    public ResponseEntity<AuthoritySearchResult> getGroupAuthorities(@PathVariable("zoteroGroupId") String zoteroGroupId,
+            @RequestParam(defaultValue = "0", required = false, value = "page") int page,
+            @RequestParam(defaultValue = "10", required = false, value = "pageSize") int pageSize,
+            @RequestParam("firstName") String firstName, @RequestParam("lastName") String lastName) {
+        
+        AuthoritySearchResult authorityResult = new AuthoritySearchResult();
```
Comment (jdamerow) [inline citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityEntryController.java:73]: define where needed

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
`auth/authority/list` just shows an empty page

**Comment by MaulikLim** [comment]
For me it throws following error in logs. But when I rebuild the project with "mvn clean install" command, it starts working. I assumed it must be a dirty build issue. I did try to solve the following issue using several options, but it didn't work.

SEVERE: Servlet.service() for servlet [appServlet] in context with path [/citesphere] threw exception [Request processing failed; nested exception is org.thymeleaf.exceptions.TemplateInputException: An error happened during template parsing (template: "ServletContext resource [/WEB-INF/views/auth/authorities/list.html]")] with root cause
org.thymeleaf.exceptions.TemplateProcessingException: Could not parse as expression: "https://localhost:8080/citesphere/auth/authority/" (template: "auth/authorities/list" - line 90, col 12)
at org.thymeleaf.standard.expression.StandardExpressionParser.parseExpression(StandardExpressionParser.java:131)
at org.thymeleaf.standard.expression.StandardExpressionParser.parseExpression(StandardExpressionParser.java:62)
at org.thymeleaf.standard.expression.StandardExpressionParser.parseExpression(StandardExpressionParser.java:44)

Once I've cleaned it, it loads the page correctly.

**Pull request closed without merging**